packages/types/src/preferences.ts

```diff
@@ -1,6 +1,6 @@
 import type { BannersState, Editor, MajorVersionWelcomeDismissed } from '.'
 
-type NotifyWhenRunCompletes = 'passed' | 'failed' | 'canceled' | 'errored'
+export type NotifyWhenRunCompletes = 'passed' | 'failed' | 'cancelled' | 'errored'
```

Comment:
Same comment here about spelling of `canceled` vs `canclled`

Reply:
We may need to opt for `cancelled`, since this is what the Cloud is sending and it's not at all practical to change their code at this point. We've got a mix of both now, which isn't really ideal.

Reply:
I think we should get feature completed and we can figure out the `canceled` vs `cancelled` stuff at the end, it's definitely confusing, we need to pick one and stick with it, I'd recommend whatever has the least risk, which is likely conforming with the Cloud's definition (double `l`).